crates/eframe/src/native/run.rs

```diff
@@ -22,7 +22,17 @@ pub use epi::NativeOptions;
 #[derive(Debug)]
 enum EventResult {
     Wait,
+    /// Causes a synchronous repaint inside the event handler. This should only
+    /// be used in special situations if the window must be repainted while
+    /// handling a specific event. This occurs on Windows when handling resizes.
+    ///
+    /// `RepaintAsap` introduces a one-frame delay, and should therefore only be
+    /// used for extremely urgent repaints.
     RepaintAsap,
```

Comment:
with this change, maybe `RepaintNow` is a more apt name?

Reply:
According to the doc, wouldn't be `RepaintNextFrame` the best?

Reply:
Or `RepaintThisFrame`, if that is what it's doing

Reply:
@Mingun This isn't "repaint this frame", that's what RepaintNext does (multiple events are handled before the next frame). This is "repaint **now**". Synchronously, immediately, before the event loop continues. It holds up all subsequent events in the loop, so in effect, it produces a new frame.